package.json

```diff
@@ -109,6 +109,7 @@
     "postcss": "8.4.31",
     "prettier": "3.2.4",
     "prettier-plugin-tailwindcss": "0.5.11",
+    "query-string": "^8.1.0",
```

Comment:
Couldn't we use the `URLSearchParams` API instead?

Reply:
I'll change it.